lib/storage.js

```diff
@@ -58,21 +58,25 @@ async function putRendition(rendition) {
     // Note: validation has happened in validate.js before
 
     if (process.env.WORKER_TEST_MODE) {
+        let sourcePath = rendition.path;
         if(rendition.name.includes('post-')) {
             // to verify post-processing renditions, since their names contain `post-` in the beginning
             // and asset-compute-cli command run-worker only looks at the file named `rendition*.*`
             rendition.name = rendition.name.substring(5);
             
             const newPath = path.join(rendition.directory, rendition.name);
             await fs.rename( rendition.path, newPath);
+            
+            sourcePath = newPath;
```

Comment:
I would avoid the term "source" here as to not confuse it with source files for a worker.

We could simply do `rendition.path = newPath` here. Would save the `sourcePath` var and all the changes below.

Reply:
I found it more confusing with the 2 reassignments. 
But changed it since you think it's not worse than what is currently used.